readme.md

```diff
@@ -102,24 +99,22 @@ mbo.finalize().then(function(mediaBlobs) {
 ```
 
 ### Split Method
```

Comment:
How does split() differ from the existing slice() method on blobs?

Reply:
Answer: it uses times instead of bytes.

Reply:
`Blob.slice()` returns a single new Blob object whereas `MediaBlobOperation.split()` operation  will return two MediaBlobs (after finalize()).

I would say `Blob.slice()` is similar to `MediaBlobOperation.trim()`. The result of both methods is a subset of the data they are called upon from start till the end.